src/polymer-project.ts

```diff
@@ -112,7 +114,11 @@ export class PolymerProject {
   bundler: Bundler;
 
   constructor(options?: ProjectOptions) {
-    this.root = options.root || process.cwd();
+    this.root = process.cwd();
```

Comment:
TIL `path.resolve()` will resolve from the current working directory by default!